src/aiogram_dialog/widgets/kbd/state.py

```diff
@@ -41,6 +43,7 @@ def __init__(
             state: State,
             on_click: Optional[OnClick] = None,
             when: WhenCondition = None,
+            show_mode: ShowMode = None,
```

Comment:
`None` is invalid value for type hint `ShowMode`

Reply:
fixed